pallets/liquid-staking-two/src/lib.rs

```diff
@@ -146,6 +146,12 @@ pub mod pallet {
     pub enum Error<T> {
         /// ExchangeRate is invalid
         InvalidExchangeRate,
+        /// Operation is not allowed
+        OperationNotAllowed,
+        /// Caller is not approved.
+        IllegalAgent,
+        /// Slash or reword event is recored.
+        EventRecorded,
```

Comment:
Can we rename it? It doesn't look like error message. @yz89 What do you suggest?